packages/tools/cookbook/src/utils/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './account-key';
+export * from './api-host';
+export * from './poll-transactions';
+export * from './print-local';
+export * from './print-sig-data';
```

Comment:
We probably don't want to export any of these functions as the cookbook will not be used as a library?

And if we want to offer these functions, they should be exposed from different libs

Reply:
The intention was not to offer this functions; I was just exporting them for ease of use within the repository. Is there a reason why we wouldn't want to export them this way?

Reply:
ahh i see 👍